tsconfig.paths.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "@functions/*": ["src/functions/*"],
+      "@libs/*": ["src/libs/*"],
+      "@models/*": ["src/models/*"]
+    }
+  }
```

Comment:
Why not have this on `tsconfig.json`?

Reply:
Once again, this was a decision by the template builder that I kept.
On `tsconfig.json` the first property is `"extends": "./tsconfig.paths.json"`

The only benefit seems to be modularizing the `tsconfig` file and avoiding cluttering it with the many arbitrary aliases that may be declared there.